quickwit/quickwit-indexing/Cargo.toml

```diff
@@ -101,6 +101,7 @@ testsuite = [
   "quickwit-storage/testsuite"
 ]
 vrl = ["dep:vrl", "quickwit-config/vrl"]
+ci-test = []
```

Comment:
I don't think this feature is ever used, but some tests depends on it. We might want to enable it somewhere